### ISSUE TITLE
How to lint *.ts and *.tsx files?

### ISSUE THREAD

**Issue by ssilvert** [joakim-eriksson/nb-eslint#26]
I have eslint set up for my TypeScript project but I can't get nb-eslint to recognize those files.  I've tried hacking my eslint.cmd but this doesn't seem to help.

How can I tell nb-eslint to look for &#8727;.ts/&#8727;.tsx instead of *.js?

**Comment by Chris2011** [comment]
Could it be, that there is a plugin for eslint? Otherwise afaik the mimetype text/javascript is hardcoded inside of the code. So atm there is no option to check other files than js.

**Comment by Chris2011** [comment]
You can find code here: https://github.com/joakim-eriksson/nb-eslint/blob/f4d712771b803955e06744b6b58ab4c721c46452/src/se/jocke/nb/eslint/task/ESLintTaskScanner.java#L122-L124 and here: https://github.com/joakim-eriksson/nb-eslint/blob/521062341413f95a07326d5b2bdb69cb973cb880/src/se/jocke/nb/eslint/annotation/ESLintAnnotationProvider.java#L44

I don't know whether eslint can handle ts files out of the box. If yes, this plugin needs to check JS/JSX and TS/TSX files.

**Comment by ssilvert** [comment]
Not a plugin.  I'm using the @typescript-eslint/parser.

Looks like nb-eslint needs an option to scan other file types.  Would you accept a PR for that?

**Comment by Chris2011** [comment]
Sure, that would be awesome :)

**Comment by ssilvert** [comment]
This is the first time I've worked on a NetBeans plugin.  I took a crack at this, and actually got it sort of working, but I ran into something very weird.   Changes I make to ESLintTaskScanner.java are no longer reflected when I run the project.

The super-weird part is that changes to ESLintAnnotationProvider.java work just fine.

I'm not sure what NetBeans is doing behind the scenes.  It's just very odd.
